virtio-devices/src/block.rs

```diff
@@ -144,11 +144,25 @@ struct BlockEpollHandler {
     inflight_requests: VecDeque<(u16, Request)>,
     rate_limiter: Option<RateLimiterGroupHandle>,
     access_platform: Option<Arc<dyn AccessPlatform>>,
-    read_only: bool,
     host_cpus: Option<Vec<usize>>,
+    acked_features: u64,
+}
+
+fn has_feature(features: u64, feature_flag: u64) -> bool {
+    (features & (1u64 << feature_flag)) != 0
```

Comment:
nit:

```suggestion
    (features & (1 << feature_flag)) != 0
```

You can leave it as it, but this makes things harder to read than necessary

Reply:
> BTW, you can make multiple comments at once instead of making one at a time, which will save a lot of time. :)

Thanks! I'm used to the one-comment-at-a-time approach. I will try to put more focus on the review feature to submit all my comments in a batch in the future!